### ISSUE TITLE
Navbar Improvements

### ISSUE THREAD

**Issue by CharlyMartin** [fidlabs/filecoin-plus-webapp-next#10]
![CleanShot 2024-12-04 at 14 36 51@2x](https://github.com/user-attachments/assets/55f31c35-8e8c-4d6d-8c89-617efc1dcf8a)

## UX
- [x] Consider reducing the navbar height to create more room for the main content, or take advantage of the current unused space to give more context about the information being presented.

## a11y
- [x] Augment the color contrast to ensure readability (Current color contrasts fail accessibility checks).
- [x] Add an `aria-label` to the link wrapping the logo to make it accessible to users relying on screen readers. Same applies to the hamburger icon button.

**Comment by tplocic20** [comment]
@CharlyMartin Is the left side of the gradient passing the contrast check? If so I can remove the gradient completely

**Comment by CharlyMartin** [comment]
@tplocic20 None of them pass the color contrast check.

![CleanShot 2024-12-04 at 15 09 44@2x](https://github.com/user-attachments/assets/00f2417b-d895-4177-a59f-36e43032e43a)

This is a screenshot from the Firefox accessibility tools. A Lighthouse audit confirms it.

Filipa, our designer, proposed this gradient: #005A9E to #0091FF

What do you think?

**Comment by tplocic20** [comment]
I can do such gradient. not a problem

**Comment by tplocic20** [comment]
@CharlyMartin I would not reduce navbar height as this spce is used as info palce on other pages. 
So it's a bit wasted on dashboards page - but I'd rather keep the UI consistent between pages
